torch/distributed/nn/api/remote_module.py

```diff
@@ -50,6 +57,17 @@ def __init__(
         The arguments of ``forward_async`` and ``forward`` are the same as
         the ``forward`` method of the module returned by the ``module_cls``.
 
+        Apart from ``forward_async`` and ``forward``, no other methods are supported from nn.Module for now.
+
+        Particularly, to create a hybrid model, typically the local modules should be
+        created outside of remote modules, rather than as submodules of any remote module (by calling ``add_module``).
+        Hybrid Example:
+        class HybridModel(nn.Module):
+            def __init__(self):
+                nn.Module.__init__(self)
+                self.remote_embedding = RemoteModule(...)
+                self.local_linear = nn.Linear(...)
+
```

Comment:
nit: You probably want to put this into a codeblock (ex: https://github.com/pytorch/pytorch/blob/master/torch/distributed/rpc/api.py#L257) so that it shows up in a code block appropriately in our docs.

Reply:
Great advice! Thx!